File: js/problem-1007.js
// https://www.beecrowd.com.br/judge/en/problems/view/1007

let input = require('fs').readFileSync('/dev/stdin', 'utf8');
let [a, b, c, d] = input.split('\n').map(value => Number(value));
let difference = (a * b - c * d);
console.log(`DIFERENCA = ${difference}`);

File: cSharp/problem-1007.cs
using System; 

class URI {

    static void Main(string[] args) { 

        int a = int.Parse(Console.ReadLine()),
        b = int.Parse(Console.ReadLine()),
        c = int.Parse(Console.ReadLine()),
        d = int.Parse(Console.ReadLine()),
        difference = a * b - c * d;
        Console.WriteLine($"DIFERENCA = {difference}");

    }

}